wpilibc/src/main/native/cpp/Drive/KilloughDrive.cpp

```diff
@@ -68,14 +67,18 @@ KilloughDrive::KilloughDrive(SpeedController& leftMotor,
 /**
  * Drive method for Killough platform.
  *
- * @param x         The speed that the robot should drive in the X direction.
- *                  [-1.0..1.0]
- * @param y         The speed that the robot should drive in the Y direction.
- *                  [-1.0..1.0]
- * @param rotation  The rate of rotation for the robot that is completely
- *                  independent of the translation. [-1.0..1.0]
- * @param gyroAngle The current angle reading from the gyro.  Use this to
- *                  implement field-oriented controls.
+ * Angles are measured counter-clockwise from straight ahead. The speed at which
```

Comment:
I'm confused. Angles are measured counter-clockwise, but rotation clockwise is positive?

Reply:
That's what I mentioned in chat. The mathematical convention for angles and angular rates is counter-clockwise for more positive values. Since joysticks return negative for left and positive for right, users would have to do the following: `mecanumDrive.Drive(joy.GetX(), -joy.GetY(), -joy.GetTwist(), gyro.GetAngle());` or `diffDrive.Drive(-joy.GetY(), -joy.GetX());`. Are we OK with that?

Reply:
Compared to RobotDrive, only the y axis had to be inverted, so either teams already did the other inversions or they weren't necessary.

Reply:
That sounded like you said "either they inverted them or they didn't", which is tautological. Could you elaborate?